numalogic/tools/types.py

```diff
@@ -17,11 +17,11 @@
 from numalogic.base import TorchModel, BaseThresholdModel, BaseTransformer
 
 try:
-    from redis.client import AbstractRedis
+    from redis.client import Redis
```

Comment:
Any specific reason?

Reply:
Yes. I was following what are the different clients that come from redis-py.  They expose mainly two clients:
Redis and StrictRedis
links: 
1) https://stackoverflow.com/questions/19021765/redis-py-whats-the-difference-between-strictredis-and-redis
2) https://devpress.csdn.net/python/62fd2d2ac677032930802e87.html 
3) https://docs.ros.org/en/hydro/api/redis/html/redis.client-module.html

Reply:
This is why I moved to Redis. Eventually we should move to StrictRedis